nixos/modules/programs/zs-apc-spdu-ctl.nix

```diff
@@ -0,0 +1,71 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.zs-apc-spdu-ctl;
```

Comment:
this needs to be added to `nixos/modules/module-list.nix` for it to be used without an explicit import